charts/device-plugin-operator/templates/operator.yaml

```diff
@@ -398,6 +398,16 @@ spec:
   selector:
     control-plane: controller-manager
 ---
+{{- if .Values.privateRegistry.registrySecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.privateRegistry.registrySecret }}
```

Comment:
Maybe the name of the secret shouldn't be the secret itself? I guess someone could list secrets while they can't get them?